src/java/com/zimbra/oauth/utilities/OAuth2Constants.java

```diff
@@ -67,5 +67,16 @@
     public static final String LC_OAUTH_CLIENT_REDIRECT_URI_TEMPLATE = "zm_oauth_%s_client_redirect_uri";
     public static final String LC_OAUTH_SCOPE_TEMPLATE = "zm_oauth_%s_scope";
     public static final String LC_OAUTH_IMPORT_CLASS_TEMPLATE = "zm_oauth_%s_import_class";
+    
+    
+    public static final String OAUTH_CLIENT_ID = "client_id";
+    public static final String OAUTH_CLIENT_SECRET = "client_secret";
+    public static final String OAUTH_CLIENT_REDIRECT_URI = "client_redirect_uri";
+    public static final String OAUTH_SCOPE = "scope";
+
+    public static final String APPNAME_YAHOO = "yahoo";
```

Comment:
These constants exist at `YahooConstants.CLIENT_NAME`, `GoogleConstants.CLIENT_NAME`, `FacebookConstants.CLIENT_NAME`, and `OutlookConstants.CLIENT_NAME`.